.github/workflows/sdks.yml

```diff
@@ -3,6 +3,9 @@ on:
   pull_request:
   schedule:
     - cron: '0 9 * * *'
+  push:   
+    branches: '*'
```

Comment:
I prefer to have the CI run once daily instead, please remove.

Reply:
What is the use case for that? I submit a pull if I want to make a change, and all direct pushes are tested by the daily CI run.

When would you want to randomly trigger a CI run with no changes to this repo?

Reply:
I was thinking for cases where there is a new devel or trunk release and we want to immediately test it (once the toolchain check goes back to tracking the latest changes).

Reply:
I just wait for the next daily CI run, but you can add it if you like.